client/src/pages/SignUp/SignUp.tsx

```diff
@@ -1,9 +1,57 @@
 import styles from './SignUp.module.scss';
+import * as React from 'react';
+
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import ArrowBackIosIcon from '@mui/icons-material/ArrowBackIos';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import TextField from '@/components/TextField/TextField'
```

Comment:
This isnt an mui, this is ours but it can also be imported like this:
import {TextField} from '@/components/'